Documentation/gettingstarted/encryption-wireguard.rst

```diff
@@ -12,6 +12,8 @@ WireGuard Transparent Encryption
 
 This guide explains how to configure Cilium with transparent encryption of
 traffic between Cilium-managed endpoints using `WireGuard® <https://www.wireguard.com/>`_.
+Aside from this guide, you can also watch this video: `eCHO episode 3: Wireguard <https://www.youtube.com/watch?v=-awkPi3D60E&t=475s>`__ on how 
+Wireguard can be used to encrypt network traffic. 
```

Comment:
“WireGuard”. It's important. CI should complain about the case here, not sure why it didn't :thinking:. Please also adjust in the video title.